ci_feature/main.sh

```diff
@@ -252,8 +308,15 @@ dpkg -l | grep docker-cimprov | awk '{print $2 " " $3}'
 
 #telegraf & fluentbit requirements
 if [ ! -e "/etc/config/kube.conf" ]; then
-      /opt/td-agent-bit/bin/td-agent-bit -c /etc/opt/microsoft/docker-cimprov/td-agent-bit.conf -e /opt/td-agent-bit/bin/out_oms.so &
-      telegrafConfFile="/etc/opt/microsoft/docker-cimprov/telegraf.conf"
+      if [ "$CONTAINER_RUN_TIME" == "docker" ]; then
```

Comment:
convert the runtime env to case-insentitive, so all consumers dont have to worry about casing.

Reply:
done.